lib/generators/custom_attributes/templates/custom_attributes_concern.rb

```diff
@@ -12,6 +12,7 @@ def add_custom_attributes_with_default_values
     return if self.custom_attributes.blank?
     self.custom_attributes.each do |cav|
       unless cav.custom_attribute_defn.default_value.blank?
+        cav.apply_default_value
```

Comment:
change the condition
if cav.custom_attribute_defn.default_value.present? && cav.value.blank?